cmd/util.go

```diff
@@ -123,25 +124,61 @@ func verifyBin(binPath ...string) {
 	}
 }
 
-func verifyKubeVersion(b string) {
+func verifyKubeVersion(major string, minor string) {
 	// These executables might not be on the user's path.
-	// TODO! Check the version number using kubectl, which is more likely to be on the path.
 
-	_, err := exec.LookPath(b)
+	_, err := exec.LookPath("kubectl")
 	if err != nil {
 		continueWithError(err, sprintlnWarn("Kubernetes version check skipped"))
 		return
 	}
 
-	cmd := exec.Command(b, "--version")
+	cmd := exec.Command("kubectl", "version")
 	out, err := cmd.Output()
```

Comment:
@lizrice Can we make kubectl version command's output only be printed when log level is set to 1? I think it clutters the output of kube-bench.